src/components/ProfileCard/ProfileCard.tsx

```diff
@@ -124,7 +125,12 @@ const ProfileCardInner = ({
   const hideUserBio =
     (userBioTextLength > 93 && skillsLength > 2) || userBioTextLength > 270;
 
-  const contributions = useContributions(user.address);
+  const epochs = useSelectedCircle().circleEpochsStatus;
```

Comment:
i would flag this use of `useSelectedCircle`, but i guess ProfileCard is going away once the new allocation flow is done anyway

Reply:
I was gonna figure out how to parse the URL and query the circle Id, but I figured this was 1) going away and 2) most import to just get this call out of the `useContributions` hook.